bedrock/firefox/templates/firefox/home.html

```diff
@@ -363,7 +363,7 @@ <h2 class="section-title">{{ _('Sync your devices') }}</h2>
       </div>
 
       <div class="sync-section-image">
-        {{ high_res_img('firefox/hub/quantum/sync.png', {'alt': _('Image of Gif shared from desktop to mobile using the Send Tab feature.'), 'width': '788', 'height': '571'}) }}
+        {{ high_res_img('firefox/home/sync.png', {'alt': _('Image of Gif shared from desktop to mobile using the Send Tab feature.'), 'width': '788', 'height': '571'}) }}
       </div>
     </div>
   </section>
```

Comment:
Looks like we can get rid of the `l10n_has_tag('firefox-home-copy-update-20171114')` below, as it's at [99% coverage](https://l10n.mozilla-community.org/langchecker/?website=0&file=firefox/hub/home-quantum.lang&action=translate&show#string51).